talk/basicconcepts/assert.tex

```diff
@@ -86,9 +86,10 @@
   \scriptsize
   \begin{Verbatim}[commandchars=\\\{\}]
 a.cpp: In function 'double f(UserType)':
-\textcolor{blue}{a.cpp:3:9:} \textcolor{red}{error:} \textcolor{blue}{static assertion failed: This function expects floating-point types.}
+\textcolor{blue}{a.cpp:3:9:} \textcolor{red}{error:} \textcolor{blue}{static assertion failed: This function}
+                                           \textcolor{blue}{expects floating-point types.}
    2 | static_assert(
      |   \textcolor{red}{std::is_floating_point_v<UserType>},
-     |   \textcolor{red}{~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~}
+     |   \textcolor{red}{~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~}
```

Comment:
My compiler really points to the `i` using a `^`. If you remove it, you can remove the line with the `~~~` as well.

Reply:
I will add it back, I thought this might have been a typo.